docs/t-sql/functions/replace-transact-sql.md

```diff
@@ -92,6 +92,28 @@ This is a desk
 (1 row(s) affected)  
 ```  
 
+The following example calculates the number of spaces in a sentence using the `REPLACE` function. First, it calculates the length of the sentence with the `LEN` function. It then replaces the ' ' characters with '' with REPLACE . After this process, it calculates the length of the sentence again. The resulting difference is the number of space characters in the sentence.
+
+```sql  
+DECLARE @str NVARCHAR(100), @len1 INT, @len2 INT
+SET @str = 'This is a sentence with a space in it.'
+SET @len1 = LEN(@str)
+SET @str = REPLACE(@str, ' ', '')
+SET @len2 = LEN(@str)
+SELECT 'Number of spaces in the sentence:' + CONVERT(NVARCHAR, @len1 - @len2)
+GO  
+```  
+
+
+ [!INCLUDE[ssResult](../../includes/ssresult-md.md)]  
+  
+```  
+------------  
+Number of spaces in the sentence:8  

```

Comment:
Adding a space after the colon to be consistent with suggested change in the code above.

```suggestion
Number of spaces in the sentence: 8  

```